doc/install.rst

```diff
@@ -32,6 +35,16 @@ Installation From Source
     ./configure --prefix "$(dirname $(pwd))" && make && make install
     popd
 
+If you want to connect to `SSL` enabled databases you need to configure `FreeTDS` with the `--with-openssl` option. OpenSSL is
```

Comment:
I would move "OpenSSL is" to the next line. In general I try to limit text to a width of 80 characters.